frontend/pulse3d/src/components/table/UploadsSubTable.js

```diff
@@ -0,0 +1,71 @@
+import { useState, memo } from "react";
+import styled from "styled-components";
+
+const SubContainer = styled.div`
+  display: flex;
+  justify-content: space-around;
+`;
+const SubHeader = styled.div`
+  display: flex;
+  background-color: var(--dark-blue);
+  color: white;
+  margin: 0 1rem;
+  padding: 0.4rem;
+  justify-content: space-around;
+`;
+const SubRow = styled.div`
+  font-size: 0.75rem;
+  width: 20%;
+  text-align: center;
+  margin: 1rem 0;
+`;
+
+export default memo(function UploadsSubTable(props) {
+  const [isChecked, setIsChecked] = useState(props.jobs.map((job) => job.checked));
+  const rows = props.jobs.map((job, idx) => {
+    let paramsString = [];
+    Object.keys(job.analysisParams).forEach((param) => {
+      if (job.analysisParams[param] !== null) {
+        if (param === "peaks_valleys") {
+          paramsString.push(<div key={Math.random()}>{`${param} : user set`}</div>);
+          return;
+        }
+        paramsString.push(<div key={Math.random()}>{`${param} : ${job.analysisParams[param]}`}</div>);
+      }
+    });
+
+    return (
+      <SubContainer key={Math.random()}>
+        <input
+          type="checkbox"
+          checked={isChecked[idx]}
+          onChange={() => {
+            if (!isChecked[idx]) {
```

Comment:
The only difference between the if/else branches here is `action: "add"/"remove"`. You can simplify this a lot by just doing
```
props.setJobToEdit({ id: job.jobId, action: isChecked[idx] ? "remove" : "add" });
setIsChecked(isChecked.map((checked, index) => (idx === index ? !checked : checked)));
```